docs/userguide/configuring.rst

```diff
@@ -113,6 +115,9 @@ In addition, examples for some specific configurations follow.
 .. [*] 8000 nodes with 32 workers each totalling 256000 workers is the maximum scale at which
        we've tested the `ExtremeScaleExecutor`.
 
+.. [*] The maximum number of nodes tested for the `WorkQueueExecutor` is 10000 GPU cores and 
+       20000 CPU cores.
```

Comment:
What does it mean to test on GPU cores? Did you have some specific GPU test code?

Reply:
The example Work Queue "task" for this test utilized GPU-specific code that interacted with the GPU. Work Queue handled the transfer of data and information from the user to the machine with the GPU, but yes, it was the example task program that had the GPU-specific test code.